code/__DEFINES/is_helpers.dm

```diff
@@ -50,6 +50,8 @@
 
 #define ismachinery(A) (istype(A, /obj/machinery))
 
+#define isairlock(A) (istype(A, /obj/machinery/door))
+
```

Comment:
I mean technically, mineral doors aren't airlocks...

Reply:
Oh god mineral airlocks exist, I'll have to fix that